src/project.coffee

```diff
@@ -282,7 +287,9 @@ class Project extends Model
   # Returns a promise that resolves to the {TextBuffer}.
   buildBuffer: (absoluteFilePath) ->
     if fs.getSizeSync(absoluteFilePath) >= 2 * 1048576 # 2MB
-      throw new Error("Atom can only handle files < 2MB for now.")
+      error = new Error("Atom can only handle files < 2MB for now.")
+      error.name = 'OversizeFileError'
```

Comment:
What do you think about just using a custom `code` instead? To be consistent with other IO errors from node.

This could be be like `error.code = 'ETOOLARGETOOPEN` or something.